app/src/main/java/sdp/moneyrun/ui/menu/LeaderboardActivity.java

```diff
@@ -91,8 +107,9 @@ public void addPlayer(Player player){
      *
      *  Initializes the player object private instance
      */
+    @RequiresApi(api = Build.VERSION_CODES.N)
     public void setMainPlayer(Player player){
-        if(user == null && player != null){
+        if(user != null || player != null){
```

Comment:
So it's possible to set the main player as null? Is this Intended?

Reply:
Oh, no this shouldn't be the case I will fix this asap, thanks!